backend/community/accounts/permissions.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework import permissions
+
+
+class IsOwnerOrCreateOnly(permissions.BasePermission):
+    """
+    Object-level permission to only allow owners of an object to edit it.
+    Assumes the model instance has an `owner` attribute.
```

Comment:
Where is this assumption of the `owner` attribute? It does not look like `return obj == request.user` is using it.